crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1860,10 +1880,17 @@ impl<'db> ClassType<'db> {
     }
 
     pub(crate) fn inherited_class_member(self, db: &'db dyn Db, name: &str) -> Symbol<'db> {
-        for base in self.bases(db) {
-            let member = base.member(db, name);
-            if !member.is_unbound() {
-                return member;
+        for superclass in self.mro(db).elements().skip(1) {
```

Comment:
how about `direct_bases`? I think I read this snippet twice and was both times wondering why there's a skip 1

Reply:
Again... `direct_bases` would be the _opposite_ of what this proposed method would do. It would iterate through _all_ proper superclasses of the class, _not_ just the direct bases :-)

Reply:
I fixed this by deleting the method ;)

Reply:
I would be OK with `proper_superclasses()`. I don't care if it's long. The goal of adding a method here  is semantic clarity, not concision. Micha's issue here was not "`elements().skip(1)` is too long", it was "it's not self-evident why we are skipping 1" -- and I agree with that.

We could also go less jargony and just call it `without_self()`?

Reply:
But, inlining it leads to a large perf regression because we now need to resolve the MRO for many more attribute lookups 😆